### PR TITLE
remove travis badges

### PR DESCRIPTION
**Pull request by gidsg**
Remove travis badges and related coverage dependency as these are no longer used﻿

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,3 @@
-# GDS metrics for Python apps [![Build Status](https://travis-ci.org/alphagov/gds_metrics_python.svg?branch=master)](https://travis-ci.org/alphagov/gds_metrics_python)
```
Comment (lfdebrux) [inline README.md:1]: We should still keep the h1

```suggestion
# GDS metrics for Python apps
```

Comment (gidsg) [reply README.md:1]: Oops 🤦

**Comment by lfdebrux** [comment]
Can we also replace Travis for the checks?

**Comment by gidsg** [comment]
@lfdebrux re: coverage this would require us to sign up for a Coveralls API key which we haven't decided to do currently. Re: build status badge, we could add it as per https://docs.github.com/en/free-pro-team@latest/actions/managing-workflow-runs/adding-a-workflow-status-badge but I'm not sure if it's of value.

**Comment by gidsg** [comment]
@lfdebrux It looks like we are still using travis, so I'll close this for now.

**Pull request closed without merging**